pageserver/src/page_service.rs

```diff
@@ -625,6 +632,22 @@ impl postgres_backend::Handler for PageServerHandler {
             // important because psycopg2 executes "SET datestyle TO 'ISO'"
             // on connect
             pgb.write_message_noflush(&BeMessage::CommandComplete(b"SELECT 1"))?;
+        } else if query_string.starts_with("tenant_shutdown") {
```

Comment:
Consider using http api. As soon as console changes to support http api this command parsing is expected to be removed

Reply:
Good point.
I removed it from PR, just to concentrate on the full pageserver shutdown. But it should be quite simple to bring it back later.